ITP/symmetry-breaking.tex

```diff
@@ -1,32 +1,50 @@
-\emph{Symmetry breaking} plays a key role in modern SAT-solving by substantially reducing the search space of assignments to a formula~\cite{biereHandbookSatisfiabilityVolume2009,Crawford}.
-For example, if one proves that all satisfying assignments to a formula $\phi$ have either (i) $x_1 = 0, x_2 = 1$, or  (ii) $x_1 = 1, x_2 = 0$, and there is a bijection between satisfying assignments of form (i) and satisfying assignments of form (ii), then one can assume, \emph{without loss of generality}, that $x_1 = 0, x_2 = 1$, and thus add unit clauses $\ov{x_1}$ and $x_2$ to the formula $\phi$ while preserving its satisfiability.
+\emph{Symmetry breaking} plays a key role in SAT solving by reducing the search space of satisfying assignments for a formula~\cite{biereHandbookSatisfiabilityVolume2009,Crawford},
+thus making a wider range of formulas practical to solve.
+For example, if one proves that all satisfying assignments to a formula $\phi$ have either (i) $x_1 = 0, x_2 = 1$, or  (ii) $x_1 = 1, x_2 = 0$, and there is a bijection between satisfying assignments of forms (i) and (ii),
+then one can assume, \emph{without loss of generality}, that $x_1 = 0, x_2 = 1$, and thus add unit clauses $\ov{x_1}$ and $x_2$ to the formula $\phi$ while preserving its satisfiability.
+There are several techniques that can automatically find symmetry-breaking clauses,
+such as structured bounded variable addition~\cite{sbva},
+but it is accepted wisdom in the SAT-solving community that problem-specific symmetry breaking is more effective.
 
-In the context of the Empty Hexagon Number, the symmetry breaking done by Heule and Scheucher consists in assuming
-that in order to search for a list of $30$ points in general position without a $6$-hole,
-it suffices to can search only amongst lists of $30$ points in \emph{canonical} position.
-These are defined as follows.
+The symmetry breaking performed by Heule and Scheucher for the Empty Hexagon Number places the points in \emph{canonical position},
+which enforces several notions of order.
+By proving that for any list of points in general position,
```

Comment:
don't love how this sentence reads. maybe instead something like:
"For any list of points in general position, there exists a list of points in canonical position with the same orientations. (something about how this is an orientation property)"

Reply:
Fair enough. I incorporated your suggestion.